.github/workflows/commit-lint.yml

```
@@ -0,0 +1,21 @@
+name: "Commit messages adheres to guidelines at https://goo.gl/p2fr5Q"
+
+on: [pull_request]
+
+jobs:
+  lint-commit-message:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          # Last 100 commits should be enough for a PR
+          fetch-depth: 100
```

I'm unsure if this syntax is valid YAML, but you can (and should IMO) use the actual number of commits.
```suggestion
          fetch-depth: ${{ github.event.pull_request.commits }} + 1
```